.pylintrc

```diff
@@ -0,0 +1,265 @@
+[MASTER]
+
+# Specify a configuration file.
+#rcfile=
+
+# Python code to execute, usually for sys.path manipulation such as
+# pygtk.require().
+#init-hook=
+
+# Profiled execution.
+profile=no
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=migrations
+
+# Pickle collected data for later comparisons.
+persistent=yes
+
+# List of plugins (as comma separated values of python modules names) to load,
+# usually to register additional checkers.
+load-plugins=
+
+
+[MESSAGES CONTROL]
+
+# Enable the message, report, category or checker with the given id(s). You can
+# either give multiple identifier separated by comma (,) or put this option
+# multiple time. See also the "--disable" option for examples.
+#enable=
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once).You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+# I0011     locally-disabled   (module-level pylint overrides)
+# disabling ```duplicate-code``` due to 5 lines that are the same near the top of
+# both xblock files - of course these are similar because it is a core pattern to
+# use - if we separated out into a separate file, we'd have the 5 lines in a separate
+# file and have 2 import statements within each file to get back the stuff - pretty
+# useless warning in this specific case
+disable=I0011,W0232,duplicate-code
```

Comment:
Why not disabling this specifically next to the code? That would allow to still catch other code duplications.

Also it would be good to replace the code names like "w0232" by their more explicit version ("no-init").